src/commands/react-native/README.md

```diff
@@ -46,6 +46,7 @@ datadog-ci react-native upload --platform android --service com.company.app --bu
 | ------------------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `--platform`        | Required  | Identifies if you are uploading iOS or Android source maps.                                                                                                                                                                                                                                                                                                     |
 | `--service`         | Required  | Set as the service name you are uploading source maps for. Datadog uses this service name to find corresponding source maps based on the `service` tag set on the RUM React Native SDK.<br>By default, the RUM React Native SDK uses your application's bundle identifier as the service.                                                                       |
+| `--bundle`          | Required  | Should be set as the path to your generated JS bundle file, `main.jsbundle` for iOS and `index.android.bundle` for Android.                                                                                                                                                                                                                                     |
```

Comment:
Could you confirm if this should be `Should` or `Must` as outlined in the RFC followed by the docs team?
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#rfc-2119
https://datatracker.ietf.org/doc/html/rfc2119